babel.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  presets: [
+    "@babel/preset-typescript",
+    "@babel/preset-react",
+    [
+      '@babel/preset-env',
+      {
+        modules: false,
+        loose: true,
+        exclude: [
+          "@babel/plugin-transform-async-to-generator",
+          "@babel/plugin-transform-regenerator",
+        ],
+      },
+    ],
+  ],
+  plugins: [
+    "babel-plugin-annotate-pure-calls",
+    "babel-plugin-dev-expression",
+    ["@babel/plugin-proposal-class-properties", { loose: true }],
+  ],
+  babelrcRoots: [
+    "packages/*"
+  ]
+};
```

Comment:
This babel conf will be used for client and react - not sure if this is the right appraoch

Reply:
cc @sibelius :)

Reply:
I would rather go with multiple babel configs that you could share some common configs

it depends on what you wanna achieve with them

Reply:
I have the root one which is for two browser packages and then the exception is node where I have node 12 target.

The question is that I don't know what browser support is kosher to target nowadays